TODO.md

```diff
@@ -0,0 +1,13 @@
+- [x] Full and SPV node e2e regtest setup with time drifts
```

Comment:
This text would be helpful in the PR description

Reply:
Put it here for now so that it's included in git and comes with the various advantages; closer to code, works offline, versioned, can directly comment in review and etc. Once this is further along, there will probably be another todo list that can be added to the description if there are any blockers in need of more viability.